src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Import/ImportType.php

```diff
@@ -64,17 +70,62 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('file', FileType::class, [
                 'required' => false,
+                'label' => $this->trans('Select a file to import', 'Admin.Advparameters.Feature'),
             ])
             ->add('iso_lang', ChoiceType::class, [
+                'required' => true,
                 'choices' => $this->getLocaleChoices(),
+                'label' => $this->trans('Language of the file', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('The locale must be installed', 'Admin.Advparameters.Notification'),
+            ])
+            ->add('separator', TextType::class, [
+                'label' => $this->trans('Field separator', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('e.g. ', 'Admin.Advparameters.Help') . ' 1; Blouse; 129.90; 5',
+            ])
+            ->add('multiple_value_separator', TextType::class, [
+                'label' => $this->trans('Multiple value separator', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('e.g. ', 'Admin.Advparameters.Help') . ' Blouse; red.jpg, blue.jpg, green.jpg; 129.90',
+            ])
+            ->add('truncate', SwitchType::class, [
+                'row_attr' => [
+                    'class' => 'js-truncate-form-group',
+                ],
+                'label' => $this->trans(
+                    'Delete all [1]categories[/1] before import',
+                    'Admin.Advparameters.Feature',
+                    [
+                        '[1]' => '<span class="js-entity-name">',
+                        '[/1]' => '</span>',
+                    ]
+                ),
+            ])
+            ->add('match_ref', SwitchType::class, [
+                'row_attr' => [
+                    'class' => 'js-match-ref-form-group',
+                ],
+                'label' => $this->trans('Use product reference as key', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('If enabled, the product\'s reference number MUST be unique!', 'Admin.Advparameters.Help'),
+            ])
+            ->add('regenerate', SwitchType::class, [
+                'row_attr' => [
+                    'class' => 'js-regenerate-form-group',
+                ],
+                'label' => $this->trans('Skip thumbnails regeneration', 'Admin.Advparameters.Feature'),
+            ])
+            ->add('forceIDs', SwitchType::class, [
+                'row_attr' => [
+                    'class' => 'js-force-ids-form-group',
+                ],
+                'label' => $this->trans('Force all ID numbers', 'Admin.Advparameters.Feature'),
+                'help' => $this->trans('If you enable this option, your imported items\' ID number will be used as is. If you do not enable this option, the imported ID numbers will be ignored, and PrestaShop will instead create auto-incremented ID numbers for all the imported items.', 'Admin.Advparameters.Help'),
```

Comment:
```suggestion
                'help' => $this->trans('Enable this option to keep your imported items’ ID number as is. Otherwise, PrestaShop will ignore them and create auto-incremented ID numbers.', 'Admin.Advparameters.Help'),
```